src/Menu.js

```diff
@@ -13,55 +14,33 @@ const Menu = ({
   noOptionsDisplay,
   className
 }) => {
-  let tabIndex = -1;
```

Comment:
the option tabIndex should be sequential, that is why it was defined outside the loop, 

- option 1 => tabIndex 0
- option 2 => tabIndex 1
- Group
  - option 1 => tabIndex 2
  - option 2 => tabIndex 3

Reply:
I see. I'll look for the best way to re-add it.

Reply:
feel free to add or suggest a different approach. initially I tried to make a little one-liner function, but the lint rules didn't like those and so the result is a fully defined `getTabIndexFunction` :|